README.md

```diff
@@ -1,101 +1,127 @@
 ```
-(venv) $ pip uninstall robot
+$ source /opt/ros/kinetic/setup.bash
+$ roslaunch rosbridge_server rosbridge_websocket.launch
+```
+### Setup [catkin workspaces](http://wiki.ros.org/catkin/conceptual_overview)
+```
+$ cd ~
+$ mkdir catkin_ws
+$ cd catkin_ws
+$ mkdir src
+$ catkin_make
+```
+You need to setup another catkin workspace in robot/rospackages
+```
+$ cd ~/Programming/robotics-prototype/robot/rospackages
+$ catkin_make
+```
+You can verify that the GUI is working by running the app.py file in basestation
+```
+$ cd ~/Programming/robotics-prototype/robot/basestation
+$ python app.py
 ```
-
-**DISCLAIMER:** This issue with module imports via `pytest` was the motivating factor to change the project directory structure. For this technique to work, the 'source' code must live inside (nested) a main directory (usually named the same as project directory name or other suitable representative identifier such as **robot** in this case). The `src` subdirectory was renamed because it made no sense when importing a package module by name like `import src.basestation.Motor`, which has no meaning/place in a module semantic context (`import robot.basestation.Motor` is much more appropriate). Most Python projects do not use a `src` directory unless it's for storing their source code that eventually gets compiled to binary (i.e. such as `.c`, `.h`, etc.. files). Also, `base-station` was renamed to `basestation` because Python no-likey dashes in import statements.
-
-Finally, to link ROS to our webpage we need to install rosbridge: `sudo apt install ros-kinetic-rosbridge-suite`.
-Before running the GUI with `./app.py`, you need to run in a separate terminal: `roslaunch rosbridge_server rosbridge_websocket.launch`. Make sure to deactivate the `venv` for this terminal (otherwise it will crash).
-Make sure it works fine: after running the `./app.py` go visit `localhost:5000` in your browser (recommended browser: Google Chrome).
-For increased modularity, make sure to add to your `.bashrc` a line to source the `.bash_aliases` file in `robot/basestation/config/` so that you can use our shortcut aliases, such as `startgui` which you should always use to start the gui. It will automatically update the environment file for the browser called `env.js`.
-
 ### .bashrc edits
-You should edit your `~/.bashrc` file so that it looks like this:
-
+You should add this to your `~/.bashrc` file.
 ```
 source /opt/ros/kinetic/setup.bash
 source ~/catkin_ws/devel/setup.bash
 # local mode
-#export ROS_MASTER_URI=http://localhost:11311
-#export ROS_HOSTNAME=localhost
+export ROS_MASTER_URI=http://localhost:11311
+export ROS_HOSTNAME=localhost
 # competition mode
-export ROS_MASTER_URI=http://172.16.1.30:11311 # competition mode
-export ROS_HOSTNAME=beep
-. /home/beep/Programming/robotics-prototype/robot/rospackages/devel/setup.bash
-. /home/beep/Programming/robotics-prototype/venv/bin/activate
+#export ROS_MASTER_URI=http://172.16.1.30:11311
+#export ROS_HOSTNAME=beep
```

Comment:
I think it is safe to replace beep everywhere in this section with $USER. What do you think @PeterGhimself

Reply:
Should be fine